oneplus/urls.py

```diff
@@ -24,13 +24,14 @@
     url(r"^signout$", oneplus.auth_views.signout, name="auth.signout"),
     url(r"^getconnected$", oneplus.auth_views.getconnected, name="auth.getconnected"),
     url(r"^a/(?P<token>\S+)$", oneplus.auth_views.autologin, name="auth.autologin"),
-    url(r"^profile", oneplus.auth_views.profile, name="auth.profile"),
-    url(r"^edit_profile", oneplus.auth_views.edit_profile, name="auth.edit_profile"),
+    url(r"^profile$", oneplus.auth_views.profile, name="auth.profile"),
+    url(r"^edit_profile$", oneplus.auth_views.edit_profile, name="auth.edit_profile"),
     url(r'^signup$', oneplus.auth_views.signup, name="auth.signup"),
-    url(r'^signup_form$', oneplus.auth_views.signup_form, name="auth.signup_form"),
+    url(r'^signup_form$$', oneplus.auth_views.signup_form, name="auth.signup_form"),
```

Comment:
That is an excellent question. As far as I know, it would serve no purpose. Removing it.